docs/management/admin/response-actions-config.asciidoc

```diff
@@ -6,8 +6,68 @@
+
+. **Generate an API access token in SentinelOne.** You'll need this token in later steps, and it allows {elastic-sec} to collect data and perform actions in SentinelOne. You must create the token with the SentinelOne permissions required for the actions you want to perform (for example, isolating and releasing an endpoint).
++
+Refer to the {integrations-docs}/sentinel_one[SentinelOne integration docs] or SentinelOne's docs for details on generating an API token.
+
+. **Install the SentinelOne integration.** The {integrations-docs}/sentinel_one[SentinelOne integration] uses the SentinelOne API to collect and ingest logs into {elastic-sec}.
++
+.Expand for details
+[%collapsible]
+====
+.. In {kib}, go to **Integrations**, search for and select **SentinelOne**, then select **Add SentinelOne**.
+.. Configure the integration with an **Integration name** and optional **Description**.
+.. Ensure that **Collect SentinelOne logs via API** is selected, and enter the required **Settings**:
+   - **URL**: The SentinelOne console URL.
+   - **API Token**: The SentinelOne API access token you generated previously.
+.. Scroll down and enter a name for the agent policy in **New agent policy name**. If other agent policies already exist, you can click the **Existing hosts** tab and select an existing policy instead. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+.. Click **Save and continue**.
+.. If you need to install {agent} on any SentinelOne-protected hosts, select *Add {agent} to your hosts* and continue with the <<enroll-agent,{agent} installation steps>>. 
++
+Select **Add {agent} later** if you want to do this later, or if all your hosts already have {agent} installed with the agent policy you just specified.
+====
+
+. **Create a SentinelOne connector.** The {kibana-ref}/action-types.html[SentinelOne connector] enables {elastic-sec} to perform actions on SentinelOne-protected hosts.
+// Update link above to sentinelone-action-type.html once that page is published.
++
+.Expand for details
+[%collapsible]
+====
+IMPORTANT: Do not create more than one SentinelOne connector.
+
+.. In {kib}, go to **Stack Management** → **Connectors**, then select **Create connector**.
+.. Select the **SentinelOne** connector.
+.. Enter the configuration information:
+   - **Connector name**: A name to identify the connector.
+   - **SentinelOne tenant URL**: The SentinelOne console URL.
+   - **API token**: The SentinelOne API access token you generated previously.
+.. Click **Save**.
+====
+
+. **Create and enable a rule to generate {elastic-sec} alerts.** Create a <<rules-ui-create,detection rule>> to generate {elastic-sec} alerts whenever SentinelOne generates alerts. Use the index pattern `logs-sentinel_one.alert*` when configuring the rule to target the data collected from SentinelOne.
```

Comment:
@natasha-moore-elastic I assume it can be any rule type as long as it searches that index pattern, but maybe we should give more specific guidance here on creating the rule. @paul-tavares @patrykkopycinski WDYT?

Reply:
Yes, it needs to be a `Detection rule (SIEM)` using a `Custom query`.
Also, in addition to to setting the source to (ONLY) `logs-sentinel_one_alert*`, they also need to enter a `custom query` value of `observer.serial_number:*`

See screen capture below

_______

<img width="1235" alt="image" src="https://github.com/elastic/security-docs/assets/56442535/2159831f-3309-46f3-95d9-384ddecc1c26">

Reply:
Revised in https://github.com/elastic/security-docs/pull/4593/commits/08062e96136b3d4d8b7958372eaec8bad5a541e1